build.sh

```diff
@@ -13,4 +13,7 @@ build_project riscv-gnu-toolchain --prefix=$RISCV
 CC= CXX= build_project riscv-pk --prefix=$RISCV/riscv64-unknown-elf --host=riscv64-unknown-elf
 build_tests
 
+mkdir -p $RISCV/riscv64-unknown-elf/share/riscv-tests
+cp -a riscv-tests/{isa,benchmarks} $RISCV/riscv64-unknown-elf/share/riscv-tests
```

Comment:
Is the {a,b} syntax a bashism?

Based on the internet it seems widely supported so I'm gonna merge for now, as this seems to be blocking travis speed ups which blocks rocket-chip development now.
"Brace Expansion, eg {a,b,c} or {1..10} is not defined by POSIX. Both forms are present in zsh, ksh93, and the first form in older ksh and mksh. The first one originated in csh, the second in zsh."